lib/ansible/modules/windows/win_domain_group_membership.py

```diff
@@ -27,6 +27,7 @@
       - A list of members to ensure are present/absent from the group.
       - The given names must be a SamAccountName of a user, group, service account, or computer.
       - For computers, you must add "$" after the name; for example, to add "Mycomputer" to a group, use "Mycomputer$" as the member.
+      - If the Object is part of another domain in a multi-domain forest, you must add the domain and "\" in front of the name.
```

Comment:
I don't believe Object should be capitalised, might be better to change it to member object as well.

Reply:
Thanks for pointing that out. changed to member object